main/main.c

```diff
@@ -715,6 +715,7 @@ PHP_INI_BEGIN()
 	STD_PHP_INI_ENTRY("output_encoding",		NULL,			PHP_INI_ALL,	OnUpdateOutputEncoding,				output_encoding,	php_core_globals, core_globals)
 	STD_PHP_INI_ENTRY("error_log",				NULL,			PHP_INI_ALL,		OnUpdateErrorLog,				error_log,				php_core_globals,	core_globals)
 	STD_PHP_INI_ENTRY("error_log_mode",			"0644",			PHP_INI_ALL,		OnUpdateLong,					error_log_mode,			php_core_globals,	core_globals)
+	STD_PHP_INI_ENTRY("error_log_usec",			"0",		PHP_INI_ALL,		OnUpdateBool,	error_log_usec,			php_core_globals,	core_globals)
```

Comment:
I would prefer unified format with microseconds. It will be a small BC break for log parsing, but on the other side we will not need endless php.ini options and every output will be the same/with higher time resolution.

Reply:
Hi @mvorisek, thank you for the comment. I actually, totally agree with you but in the @internals discussions, there were many votes not to break the compatibility. I would appreciate it if you could share your opinion as well, so that everyone is recorded: https://externals.io/message/118865